test/micro/org/openjdk/bench/java/util/stream/ops/value/SizedSum.java

```diff
@@ -26,6 +26,8 @@
 import org.openjdk.jmh.annotations.BenchmarkMode;
 import org.openjdk.jmh.annotations.Fork;
 import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Measurement;
 import org.openjdk.jmh.annotations.Mode;
 import org.openjdk.jmh.annotations.OutputTimeUnit;
 import org.openjdk.jmh.annotations.Param;
```

Comment:
Ditto imports vs annotations.

Reply:
These last changes reduce the run time to about 4h45m with adequate result stability.